### PR TITLE
[fix](cloud) cloud mode support txn load for mow tables

### PR DESCRIPTION
**Pull request by mymeiyi**

### CODE REVIEW

**Comment by mymeiyi** [comment]
run  buildall

**Comment by mymeiyi** [comment]
run buildall

**Comment by mymeiyi** [comment]
run buildall

**Comment by mymeiyi** [comment]
run buildall

**Comment by mymeiyi** [comment]
run buildall

**Comment by mymeiyi** [comment]
run buildall

**Comment by mymeiyi** [comment]
run p0

**Comment by mymeiyi** [comment]
run cloud_p0

**Comment by mymeiyi** [comment]
run buildall

File: fe/fe-common/src/main/java/org/apache/doris/common/Config.java [hunk 1]
```diff
@@ -3114,6 +3114,9 @@ public static int metaServiceRpcRetryTimes() {
     @ConfField(mutable = true, description = {"存算分离模式下calculate delete bitmap task 超时时间，默认15s"})
     public static int calculate_delete_bitmap_task_timeout_seconds = 15;
 
+    @ConfField(mutable = true, description = {"存算分离模式下事务导入calculate delete bitmap task 超时时间，默认300s"})
+    public static int calculate_delete_bitmap_task_timeout_seconds_for_transaction_load = 300;
```
Comment (dataroaring) [inline fe/fe-common/src/main/java/org/apache/doris/common/Config.java:3118]: What about using calculate_delete_bitmap_task_timeout_seconds and adjust calculate_delete_bitmap_task_timeout_seconds to a larger number.

**Review by dataroaring — DISMISSED** [review]
LGTM

**Comment by mymeiyi** [comment]
run buildall

**Comment by mymeiyi** [comment]
run external

**Comment by mymeiyi** [comment]
run p0

**Comment by mymeiyi** [comment]
run buildall

**Comment by mymeiyi** [comment]
run buildall

**Comment by mymeiyi** [comment]
run p0

**Comment by mymeiyi** [comment]
run buildall

**Comment by mymeiyi** [comment]
run buildall

**Comment by dataroaring** [comment]
run buildall

**Comment by dataroaring** [comment]
run buildall

**Review by dataroaring — DISMISSED** [review]
LGTM

File: be/src/olap/base_tablet.h [hunk 2]
```diff
@@ -236,10 +239,21 @@ class BaseTablet {
...
                                        int64_t txn_id, int64_t txn_expiration = 0);
+    // The {invisible_rowsets} and {base_txn_id} is set in transaction load. In transaction load:
+    // * the {txn_id} is sub_txn_id
+    // * the {invisible_rowsets} is used to calculate delete bitmap
+    // * the {base_txn_id} is used as lock id in cloud mode
+    // * the {next_visible_version} is the next version of the partition visible version
+    static Status update_delete_bitmap(const BaseTabletSPtr& self, TabletTxnInfo* txn_info,
+                                       int64_t txn_id, int64_t txn_expiration,
+                                       std::vector<RowsetSharedPtr>* invisible_rowsets,
+                                       int64_t base_txn_id, int64_t next_visible_version,
```
Comment (zhannngchen) [inline be/src/olap/base_tablet.h:250]: change `base_txn_id` to `lock_id`

File: regression-test/suites/insert_p2/transaction/txn_insert_concurrent_insert_mow.groovy [hunk 3]
```diff
@@ -85,8 +80,12 @@ suite("txn_insert_concurrent_insert_mow") {
         }
     }
     sql """ sync """
+    def t2_row_count = 6001215
+    def result = sql """ select count() from ${tableName}_2 """
+    logger.info("${tableName}_2 row count: ${result}, expected: ${t2_row_count}")
+    assertEquals(t2_row_count, result[0][0] as int)
 
-    def dbName = "regression_test_insert_p2"
+    def dbName = "regression_test_insert_p2_transaction"
```
Comment (zhannngchen) [inline regression-test/suites/insert_p2/transaction/txn_insert_concurrent_insert_mow.groovy:88]: Do we have any cases test sequence column?

**Comment by mymeiyi** [comment]
run buildall

**Review by zhannngchen — APPROVED** [review]
LGTM

**Review by dataroaring — APPROVED** [review]
LGTM

**Pull request merged**